src/platformdirs/unix.py

```diff
@@ -7,7 +7,9 @@
 
 from .api import PlatformDirsABC
 
-if sys.platform.startswith("linux"):  # pragma: no branch # no op check, only to please the type checker
+if (
+    sys.platform.startswith("linux") or sys.platform == "sunos5"
```

Comment:
We should add a test for this.

Reply:
Sure, but I'm not sure what kind of test you would like to see.  Please advise.  I just updated exception wording and changed test name.  Is this enough?  Thank you.

Reply:
We need the test that presumably runs on the Linux, but fails should be removed this sunos if check you are adding. You probably want to mock the platform call to achieve it.

Reply:
Sorry, I do not understand what exactly you mean by that.  Could you please elaborate?  Thank you.